docker/scripts/test.sh

```diff
@@ -0,0 +1,5 @@
+
+docker exec cl-doctor pip install x-ray
```

Comment:
Chris, if you do take a look at this PR, can you clarify why this is here? Is x-ray required to be installed in/by cl-doctor for these Selenium tests to run successfully?

Reply:
It's not. Let's remove it.